content/api/templates.apib

```diff
@@ -410,33 +188,36 @@ For a master account owned template **only**, the results will include the `shar
             }
         }
 
-## Update [/templates/{id}{?update_published}]
+### Update a Draft [PUT /templates/{id}]
 
-### Update a Template [PUT]
+Updates the draft version of a template.
 
-Update an existing template by specifying its ID in the URI path and a **template object** as the PUT request body.
-By default, an update operates on an existing draft version.
-An existing published version can be overwritten directly by setting the `update_published` query parameter to `true`.
+If a content object is provided in the request body, it must contain all relevant content fields whether they are being changed or not.
+The new content will completely overwrite the existing content.
 
-<banner status="info"><strong>Note</strong>: Attempting to update the published version of a template when only a draft version exists (and vice versa) will result in an error.</banner>
 
-The `name` field may be modified, but the `id` field is read only.
++ Parameters
+    + id (required, string, `11714265276872`) ... ID of the template
 
-If a content object is provided in the update request, it must
-contain all relevant content fields whether they are being changed or not.
-The new content will completely overwrite the existing content.
 
-The example shows an update that will rename the template, enable open tracking,
-and update the content all in one API call. All content fields are included whether they are being
-changed or not.
++ Data Structure:
```

Comment:
Extra colon.

Also, this needs to be before the `+ Parameters` for it to render.